redhat/radiusd.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=FreeRADIUS multi-protocol policy server
 After=network-online.target
-Documentation=man:radiusd(8) man:radiusd.conf(5) http://wiki.freeradius.org/ http://networkradius.com/doc/
+Documentation=man:radiusd(8) man:radiusd.conf(5) https://wiki.freeradius.org/ https://freeradius.org/documentation/freeradius-server/
```

Comment:
Unless you want **https://networkradius.com/doc/current/**?

Plain **https://networkradius.com/doc/** doesn't work.

Reply:
The freeradius documentation page is fine.  We'll move all of the content there.